src/olympia/signing/tests/test_views.py

```diff
@@ -237,7 +237,7 @@ def test_version_is_beta_unlisted(self):
         version = qs.get()
         assert version.addon.guid == self.guid
         assert version.version == version_string
-        assert version.statuses[0][1] == amo.STATUS_LITE
+        assert version.statuses[0][1] == amo.STATUS_UNREVIEWED
```

Comment:
This was another test that was using a trusted fixture (addon_3615).  Though tbh I wasn't doing addons when signing was added so I wasn't 100% what the flow was supposed to be generally for unlisted addons.